### DISCUSSION TITLE
Use httpProxy from the config.

### DISCUSSION THREAD

**Discussion by WalterliTell** [solspace/craft-freeform#1692] [Feature Requests]
We need to use a httpProxy because of our firewall. Freeform seems to ignore the httpProxy configuration.

Could you consider using Craft::createGuzzleClient() instead of new Client().

For example:
`$client = Craft::createGuzzleClient([
            'headers' => [
                'Referer' => Craft::$app->getSites()->getPrimarySite()->getBaseUrl(),
            ],
        ]);`

I'm currently adding the proxy manually in your ReCaptcha.php but that's not optimal for maintenance.
Maybe I'm missing something, if so please correct me :)

Kind regards

**Comment by kjmartens** [comment]
Thanks for your feedback @WalterliTell. We'll check into this. 🙂

**Comment by kjmartens** [comment]
This will be available in Freeform 5.9.0, available sometime next week. 🙂

**Reply by WalterliTell** [reply]
Well that was quick. Thank you very much!
Finally no more "broken" forms after updating and forgetting to set the proxy manually 😅

**Reply by kjmartens** [reply]
Happy to help @WalterliTell  🙂

**Comment by kjmartens** [comment]
This should now be resolved in Freeform 5.9. 🙂
